### DISCUSSION TITLE
"Move" needs options "Left to Right" or "Right to Left" like "Copy"

### DISCUSSION THREAD

**Discussion by redtetrahedron** [WinMerge/winmerge#1412] [Ideas]
In the folder comparison pane, when there is a "Left only" or "Right only" file, the "Move" option only has "Move to..." (which prompts for a destination folder). Frequently, I need "Move Left to Right" or "Move Right to Left". Currently I accomplish this by doing a Copy, then a Delete of the original, but it would be nice to have it in one step.

**Comment by sdottaka** [comment]
Added Move->Left to Right and Right to Left menu items to the context menu in PR https://github.com/WinMerge/winmerge/pull/1732.

![image](https://user-images.githubusercontent.com/98126/220800539-9192bca5-e08a-4a83-a510-e38197a07108.png)
